easybuild/easyconfigs/p/PCAngsd/PCAngsd-0.97-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'Tarball'
+
+name = 'PCAngsd'
+version = '0.97'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.popgen.dk/software/index.php/PCAngsd'
+description = """PCAngsd, which estimates the covariance matrix for low depth 
+NGS data in an iterative procedure based on genotype likelihoods and is able 
+to perform multiple population genetic analyses in heterogeneous populations."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/Rosemeis/pcangsd/archive/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+patches = [
+    'PCAngsd-python-hashbang.patch',
```

Comment:
@FokkeDijkstra Style nitpicking: you can make this a single line.

Also, the patch file should include the version to which it applies, so please rename the patch to something like `PCAngsd-0.97_python-hashbang.patch` (and then use `%(version)s` to avoid hardcoding the version here)

```python
patches = ['PCAngsd-%(version)s_python-hashbang.patch']

Reply:
Done.